.github/workflows/ci.yml

```
@@ -391,6 +393,54 @@ jobs:
           name: webapp-client
           path: webapp/client/
 
+
+  devolutions-gateway-players:
```

suggestion: I think we are building a project called `player`, not `players`

```suggestion
  devolutions-gateway-player:
```